swag-deploy-action/action.yaml

```diff
@@ -0,0 +1,27 @@
+name: 'Swag Deploy Action'
```

Comment:
Probably want to strip out `Swag` since it's unrelated.  It would be in the author field already?